src/azul/openapi/schema.py

```diff
@@ -22,6 +22,10 @@
 TYPE = Union[None, Type, str, JSON]
 
 
+def wrap_json_schema_content(json):
```

Comment:
I have my doubts about this. For one, it is not used for composing a schema so it's not in the right module.

I also see only three call sites so I am not sure its even worth it. Especially considering that the name of the method is half as long as its body.

But leave this as is for now. I reserve the right to refactor it.

Reply:
There are many other endpoints where it could be used, and it saves one level of indentation in cases where the JSON spec is inlined instead of using a variable. A function is also less error prone than repeatedly typing nested string/dict literals.

Reply:
Makes sense. When you introduce such an improvement you should either refactor all potential usages or create a ticket to remind us to do so. I will be doing the former in a separate commit to your branch.